deepforest/cascade.py

```diff
@@ -315,7 +316,7 @@ def _build_regressor_predictor(
 
     Parameters
     ----------
-    X : :obj:`numpy.ndarray` of shape (n_samples, n_features)
+    X : :obj:`numpy.ndarray` or `pandas.DataFrame` or List of shape (n_samples, n_features)
```

Comment:
I think there is no need to further add `or List of shape (n_samples, n_features)`

Reply:
Now the code comment is modified to `array-like` referring to xgboost and lightgbm.